dewy/main.py

```diff
@@ -54,3 +56,12 @@ async def healthcheck() -> dict[str, str]:
 
 
 app.include_router(api_router)
+
+
+# Get the directory of the current Python script
+current_file_path = Path(__file__).resolve()
+# Set the path to the directory where your React app's build is located
+react_build_path = current_file_path.parent / 'frontend' / 'dist'
+
+# Serve static files from the React app build directory
+app.mount("/admin", StaticFiles(directory=str(react_build_path), html=True), name="static")
```

Comment:
Should we make this option? I assume if we're doing development on the frontend, we'd have `vite` or something like that take care of serving the frontend so we get hot-reloading?